src/monster.cpp

```diff
@@ -97,7 +97,7 @@ const std::string& Monster::getNameDescription() const
 
 bool Monster::canSee(const Position& pos) const
 {
-	return Creature::canSee(getPosition(), pos, Map::maxClientViewportX + 1, Map::maxClientViewportX + 1);
+	return Creature::canSee(getPosition(), pos, Map::maxClientViewportX + 1, Map::maxClientViewportY + 1);
```

Comment:
For monsters, using `maxClientViewportX` seems like correct behavior. Unless Cip says otherwise.

Reply:
Good point, I will revert this line and leave it as it alone to mirror cip behavior 🤔 we could use better variable names.

Reply:
> For monsters, using `maxClientViewportX` seems like correct behavior. Unless Cip says otherwise.

Why? If Y axis is calculated, players running vertically can escape easily than horizontally.
Monster ex: can see X: 10, see Y: 8.